xchange-binance/src/main/java/org/knowm/xchange/binance/dto/account/TransfertHistoryResponse.java

```diff
@@ -0,0 +1,80 @@
+package org.knowm.xchange.binance.dto.account;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+import lombok.Data;
+
+public final class TransfertHistoryResponse
+    extends WapiResponse<List<TransfertHistoryResponse.TransfertHistory>> {
+
+  private final TransfertHistory[] transfertHistories;
+
+  public TransfertHistoryResponse(
+      @JsonProperty("transfers") TransfertHistoryResponse.TransfertHistory[] transfertHistories,
+      @JsonProperty("success") boolean success,
+      @JsonProperty("msg") String msg) {
+    super(success, msg);
+    this.transfertHistories = transfertHistories;
+  }
+
+  @Override
+  public List<TransfertHistory> getData() {
+    return Arrays.asList(transfertHistories);
+  }
+
+  @Override
+  public String toString() {
+    return "transfertHistories [rows=" + Arrays.toString(transfertHistories) + "]";
+  }
+
+  @Data
+  public static final class TransfertHistory {
+    private String from;
+    private String to;
+    private String asset;
+    private BigDecimal qty;
+    private long time;
+
+    public String getFrom() {
+      return from;
+    }
```

Comment:
if your using lombok then these setters / getters are not needed

Reply:
I haven't created unit test but if you remove getter, are you still able to create UT ?

Reply:
I'm not telling to remove getters
I'm telling to remove the getter code you have written by hand
lombok will generate getters for you - this is how the `@Data` annotation you have used here works https://projectlombok.org/features/Data

Reply:
file name has a typo

Reply:
> file name has a typo

Fixed all references from Transfert to Transfer ( without the 't' )